server/routes/v1/article.js

```diff
@@ -15,5 +20,6 @@ article.post(
 );
 article.get('/:id', validateUuidParams, getArticle);
 article.get('/', getArticles);
+article.patch('/:id', validateUuidParams, verifyUser, updateArticle);
```

Comment:
I think we could have the `checkIsVerified` middleware here, so only verified users can update an article. 👍

Reply:
Yeah, I thought about that too, but then again since each article is attached to a user id, and a user must be verified before creating an article, so if the user is not verified, there would be no article to update in the first place, and if you check that update controller, the where clause requires the userId of the creator of an article before you can update. So if a user is not verified, he would not be able to create an article, thus no article to update.